packages/paste-website/src/pages/components/file-picker/index.mdx

```diff
@@ -0,0 +1,257 @@
+---
+title: File Picker
+package: '@twilio-paste/file-picker'
+description: An input allows users to enter text on a single line.
+slug: /components/file-picker/
+---
+
+import {graphql} from 'gatsby';
+import {Label} from '@twilio-paste/label';
+import {FilePicker, FilePickerButton} from '@twilio-paste/file-picker';
+import {Anchor} from '@twilio-paste/anchor';
+import {HelpText} from '@twilio-paste/help-text';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {useUID} from '@twilio-paste/uid-library';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout';
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {
+  defaultFilePicker,
+  disabledFilePicker,
+  requiredFilePicker,
+  i18nFilePicker,
+} from '../../../component-examples/FilePickerExamples.ts';
+import Changelog from '@twilio-paste/file-picker/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/file-picker"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/file-picker/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "File Picker"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/file-picker"
+  storybookUrl="/?path=/story/components-file-picker--"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{FilePicker, FilePickerButton, Label, HelpText, useUID}} language="jsx" noInline>
+  {defaultFilePicker}
+</LivePreview>
+
+## Guidelines
+
+### About File Picker
+
+Use a File Picker when users need to upload a document in a form flow. The File Picker allows any type of file by default, but the desired file type can be specified using the <inlineCode>accept</inlineCode> prop. This is a native HTML file picker, `<input type="file" />`, so there's currently no drag-and-drop functionality. <Anchor href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file" showExternal>Read more about the native file picker on MDN.</Anchor> The input is hidden visually, but still appears in the DOM, so that we're able to apply custom styles in an accessible way.
+
```

Comment:
suggestion: I tried to pick multiple files and I couldn't, which I think is what we want. Should we point that out here, that this file picker is for one file at a time?

Reply:
I can add a note about that to the docs!